packages/framework/src/schemas/providers/push/fcm.schema.ts

```diff
@@ -26,14 +26,14 @@ const fcmOutputSchema = {
         'This parameter specifies a logical expression of conditions that determine the message target.\nSupported condition: Topic, formatted as yourTopic in topics. This value is case-insensitive.\nSupported operators: &&, ||. Maximum two operators per topic message supported.\n',
       type: 'string',
     },
-    notification_key: {
+    notificationKey: {
       description:
         'This parameter is deprecated. Instead, use **to** to specify message recipients. For more information on how to send messages to multiple devices using **to**, see [Device Group Messaging](https://firebase.google.com/docs/cloud-messaging/notifications).\n',
       type: 'string',
     },
-    collapse_key: {
+    collapseKey: {
       description:
-        'This parameter identifies a group of messages (e.g., with ```"collapse_key": "Updates Available"```) that can be collapsed, so that only the last message gets sent when delivery can be resumed. This is intended to avoid sending too many of the same messages when the device comes back online or becomes active (see **delay_while_idle**).\nNote that there is no guarantee of the order in which messages get sent.\nNote: A maximum of 4 different collapse keys is allowed at any given time. This means a FCM connection server can simultaneously store 4 different send-to-sync messages per client app. If you exceed this number, there is no guarantee which 4 collapse keys the FCM connection server will keep.\n',
+        'This parameter identifies a group of messages (e.g., with ```"collapseKey": "Updates Available"```) that can be collapsed, so that only the last message gets sent when delivery can be resumed. This is intended to avoid sending too many of the same messages when the device comes back online or becomes active (see **delay_while_idle**).\nNote that there is no guarantee of the order in which messages get sent.\nNote: A maximum of 4 different collapse keys is allowed at any given time. This means a FCM connection server can simultaneously store 4 different send-to-sync messages per client app. If you exceed this number, there is no guarantee which 4 collapse keys the FCM connection server will keep.\n',
```

Comment:
Also updating the descriptions here to use `camelCasing`. This is important to ensure that developers following these schema descriptions have an accurate experience.